_data/people/cranmer.yml

```
@@ -510,3 +510,10 @@ presentations:
   meetingurl: https://indico.cern.ch/event/957103/
   location: "(Virtual)"
   focus-area: as
+- title: "Explorations at the Physics ∩ ML Interface"
+  date: 2021-05-05
+  url: https://www.youtube.com/watch?v=s5y-13elZrs
+  meeting: "Physics ∩ ML seminar series"
+  meetingurl: http://www.physicsmeetsml.org/posts/sem_2021_05_06/
+  location: (Virtual)
+  focus-area: as, ia
```

```suggestion
  focus-area: [as, ia]
```

This needs to be a yaml list. (Whatever form you like, inline or expanded)

---

thanks. I noticed that when the checks failed, but didn't have time to change it. I didn't notice this until I sent the updated PR below.